packages/experiments/src/components/VerticalPersona/VerticalPersona.view.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+import { IVerticalPersonaStyles } from './VerticalPersona.types';
+import { VerticalPersonaView } from './VerticalPersona.view';
+import { IProcessedStyleSet } from 'office-ui-fabric-react/lib/Styling';
+
+// These tests will ensure that your styles regions have classname representation in snapshot output.
+// (Also, classNames is a required prop for views, so we have to supply it for tests.)
+const testVerticalPersonaClassNames: IProcessedStyleSet<IVerticalPersonaStyles> = {
+  root: 'test-cn-root',
+  text: 'test-cn-text',
+  secondaryText: 'test-cn-secondaryText',
+  subComponentStyles: {}
+};
+
+// Views are just pure functions with no statefulness, which means they can get full code coverage
+//    with snapshot tests exercising permutations of the props.
```

Comment:
>     [](start = 2, length = 4)

redundant whitespace

Reply:
@JasonGore is there a reason for this extra whitespace? (It's present in the `.mustache` file too)

Reply:
No particular reason, can change it in the template easily enough.